analyzers/tests/SonarAnalyzer.TestFramework.Test/Verification/VerifierTest.cs

```diff
@@ -188,6 +188,7 @@ public class Sample
                 .WithAdditionalFilePath(AnalysisScaffolding.CreateSonarProjectConfig(TestContext, ProjectType.Unknown))
                 .VerifyNoIssueReported();
 
+        [Ignore("REVERT AFTER 9.20 RELEASE")]
```

Comment:
We releasing 9.19. Is there a reason to skip a version, or the message should rather be `REVERT AFTER 9.19 RELEASE`?
While you are changing it, I don't see the need to write the message in all capital: it won't make it any more visible.
If we want to not forget about it, I would create a card visible on the board right away.